### PR TITLE
Pendingintent alarm manager

### PR DESCRIPTION
**Pull request by Solod**
Have a look, please.
Waiting for comments

### CODE REVIEW

File: app/src/main/java/com/solodilov/evgen/pingservers/MainActivity.java [hunk 1]
```diff
@@ -1,53 +1,113 @@
...
         getSupportFragmentManager().beginTransaction()
                 .add(R.id.activity_container, MyFragment.newInstance())
                 .commit();
         setBroadcastReceiver();
+        initIntentBackground();
+    }
+
+    @Override
+    public void onStartService(String command, boolean taskServiceFragment) {
+        String mCommand = "/system/bin/ping -c 4 " + command;
```
Comment (Gwindor) [inline app/src/main/java/com/solodilov/evgen/pingservers/MainActivity.java]: add -s 1

Comment (Gwindor) [inline app/src/main/java/com/solodilov/evgen/pingservers/MainActivity.java]: extract constant

**Comment by Gwindor** [comment]
add the clear button to clear text

**Comment by Gwindor** [comment]
add automatic scroll to the bottom of the text

File: app/src/main/java/com/solodilov/evgen/pingservers/MainActivity.java (hunk 1 above)
Comment (Solod) [inline app/src/main/java/com/solodilov/evgen/pingservers/MainActivity.java]: then add the field to set the criteria

**Pull request merged**